addons/mail/static/src/components/follow_button/follow_button.js

```diff
@@ -46,8 +46,14 @@ export class FollowButton extends Component {
      * @private
      * @param {MouseEvent} ev
      */
-    _onClickUnfollow(ev) {
-        this.thread.unfollow();
+    async _onClickUnfollow(ev) {
+        await this.thread.unfollow();
+        if (this.props.chatterLocalId) {
+            const chatter = this.messaging.models['Chatter'].get(this.props.chatterLocalId);
+            if (chatter) {
+                chatter.reloadParentView({ fieldNames: ['message_follower_ids'] });
+            }
+        }
```

Comment:
Preparation for OWL 2 in 15.3 where it's not longer possible to trigger in component.

So the workaround here is to pass chatter(LocalId) to the sub-components, and let themselves call `reloadParentView`.

Also added new named parameter to `Chatter/reloadParentView()`, `fieldNames`, which let reload just this field.